contracts/axelar-interchain-token-service/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
```

Comment:
PR titles should follow this style: https://www.conventionalcommits.org/en/v1.0.0/#summary

see previous merged PRs